playbooks/roles/kubernetes/tasks/deploy-zowe-k8.yaml

```diff
@@ -0,0 +1,310 @@
+---
+  - name: Get namespace and service account files
+    delegate_to: localhost
+    find:
+      paths: ../containers/kubernetes/common/
+      file_type: file
+      recurse: yes
+    register: k8s_ns
+  - name: Apply Zowe namespace and service account 
+    delegate_to: localhost
+    kubernetes.core.k8s:
+      kubeconfig: "{{ kubeconfig }}" 
+      context: "{{ k8s_context }}"
+      state: present 
+      src: "{{ item.path }}"
+    with_items: "{{ k8s_ns.files }}"
```

Comment:
One consideration is `zowe` namespace must be created before creating everything else including `zowe-sa`. Since `namespace: zowe` is stated in `zowe-sa` manifest.

Reply:
Okay. I'll deploy them individually. Namespace first and then Service Account.